ansible/roles/cleaner/tasks/main.yml

```diff
@@ -21,10 +21,32 @@
   ignore_errors: true
   tags: dashboard
 
-- name: kill osdslet and osdsdock and osdsapiserver daemon service
-  shell: killall osdslet osdsdock osdsapiserver
-  when: install_from != "container"
+# Kill all the systemd processes and disable services
```

Comment:
This comment valid?

Reply:
Oh! That's not right. 
Good catch @kumarashit 
This should be **Kill all the SODA standalone processes** 
I will fix it in another PR when fixing some other typos.

Reply:
ok